test/unit/test_autosubmit.py

```diff
@@ -0,0 +1,34 @@
+from textwrap import dedent
+from pathlib import Path
+from autosubmit.autosubmit import Autosubmit
+from test.unit.conftest import autosubmit_config
+from autosubmitconfigparser.config.basicconfig import BasicConfig
+
+import re
+
+def test_as_conf_default_values(tmp_path, autosubmit_config):
+
+    autosubmit_config('a000', {})
+
+    ini_file = Path(f'{BasicConfig.LOCAL_ROOT_DIR}/a000/conf')
+    ini_file.mkdir(parents=True, exist_ok=True)
+    ini_file = ini_file / 'jobs_a000.yml'
+
+    with open(ini_file, 'w+') as f:
+        f.write(dedent('''\
+                DEFAULT:
+                    AUTOSUBMIT_VERSION:
+                    EXPID: "a001"
```

Comment:
This will be ignored in the configuration, as you call `autosubmit_config('a000', ...)`. Maybe we can replace this by a var... will send a diff example...